src/deadline/job_attachments/caches/hash_cache.py

```diff
@@ -45,12 +47,12 @@ class HashCache(CacheDB):
     """
 
     CACHE_NAME = "hash_cache"
-    CACHE_DB_VERSION = 2
+    CACHE_DB_VERSION = 3
```

Comment:
Does this impact the user in anyway? ie. Forcing a purge of the existing cache? Or will everything continue to work for previous users and new users alike?

Reply:
It does effectively purge the cache, but users do not have to take any action.